### ISSUE TITLE
[bug] Less than 3 characters crashes the display

### ISSUE THREAD

**Issue by caraar12345** [OctopusSmartEnergy/Home-Pro-SDK-Public#5]
If you try and display a message of less than 3 characters, it freezes the display. 

After around ~15 seconds, the display resets back to the `CAD` screen. If you don't send a new display message and then touch through to the SDK screen, it'll freeze again. Rinse and repeat.

Side note: it would be really cool to be able to see the code behind the APIs - any reason they're not available?

**Comment by awais11010** [comment]
Hi @caraar12345, thanks for the report and sorry for getting to this so late :expressionless:. This indeed sounds like a bug and I'll give this a try.

I did not get your point around seeing the code behind the APIs, would you kindly clarify a bit more on that?

**Comment by caraar12345** [comment]
No worries! Things are busy 🙇‍♂️ 


What I meant was...

Is it possible to open source the backend APIs on the pro? Or to allow ssh access outside of the docker container?

**Comment by awais11010** [comment]
Nah, I don't think that's in the plans for now. Containerized apps were the way to go when we designed the security side of things for the base platform. Out of curiosity, why would you need such access or want to have a look at the source of APIs?
